### ISSUE TITLE
Mod info overview (finish + patch)

### ISSUE THREAD

**Issue by SenorRagequit** [exorstudios/riftbreaker-tools#219]
### Enter your feature description.

I have no real idea how it could be made possible, but it would be awesome if we could have some mod info for our mods controlled by the workspace editor.

Maybe a way to set the current mod as "finished" and it brings up a menu where we can set:
- Name of the mod
- Description
- Dependencies on other mods
- Version (1.0, 1.0.1, etc)
- Mod Authors (in case more than 1)

That stuff is then inside the mod as some mod_info.txt or whatever and can later be read by the game internal mod manager and show that info.

Additionally, something most games don't offer, but I hope Exor does: **A way for patches**
Next to setting a mod as finished, you can change stuff and make it a patch.
And the game then has some mod_patch_[modname].txt file where we can write a text what changed, change the mod version etc.

Thinking about the mod manager might be far ahead, but until then, players can read the txt-files to understand what's happening.

**Comment by lukaasm** [comment]
Yes, a mod manifest will be prolly needed in the future. It should also include game version it was created against.

So when we have mod menu in-game we can show warnings for "out-of-date" mods

**Comment by lukaasm** [comment]
initial work is done, further expansion as separate issues
